glue/viewers/image/state.py

```diff
@@ -495,6 +495,7 @@ class ImageLayerState(BaseImageLayerState):
     bias = DDCProperty(0.5, docstring='A constant value that is added to the '
                                       'layer before rendering')
     cmap = DDCProperty(docstring='The colormap used to render the layer')
+    preferred_cmap = DDCProperty(docstring='The preferred colormap used to render the layer')
```

Comment:
I wouldn't make this a state property - I would just make it so that when ``self.cmap`` is set, you can check ``layer.style.preferred_cmap`` to see if it's set, and if not, use colormaps.members[0][1]

Reply:
Eliminated the `preferred_cmap` as a state property.